game-core/src/main/java/swinglib/JTextAreaBuilder.java

```diff
@@ -38,6 +40,32 @@ public static JTextAreaBuilder builder() {
     return new JTextAreaBuilder();
   }
 
+  /**
+   * Attaches a listener that is fired whenever text is typed in the textArea.
+   * Note, the listener will be notified if text is either typed or pasted.
+   *
+   * @param textArea The component to which we attach a listener.
+   * @param listener The listener that should be invoked on key events.
+   */
+  public static void addTextListener(final JTextArea textArea, final Runnable listener) {
```

Comment:
Does this need to be replaced in favor of the new `DocumentListenerBuilder#attachDocumentListener()` method?

Reply:
Yup, `DocumentListenerBuilder` was opened while this PR was in flight, nonetheless it is a good cleanup, good eye.

Done with these two updates:
- Call to this method removed in:  2c252c2 
- Code removed in: 9b5736e